dealership/contracts.py

```diff
@@ -1,12 +1,79 @@
+interest_car = 1.07
+interest_motorcycle = 1.03
+interest_truck = 1.11
+employee_discounted_price = 0.9
+lease_multiplier_car = 1.2
+lease_multiplier_motorcycle = 1.0
+lease_multiplier_truck = 1.7
+
```

Comment:
we normally create constants in caps and in it's own class .. for example, INTEREST_CAR and LEASE_MULTIPLIER_CAR should be inside `Car` class.. make sense?